### ISSUE TITLE
Dynamic tooltip text?

### ISSUE THREAD

**Issue by nijlandvpr** [ryangjchandler/alpine-tooltip#5]
Hi Ryan

Thanks for this - it is great.

Question: Is there a way to make the text of the tooltip dynamic? I'm trying to figure out how to make it toggle between "Enable" and "Disable" based on an underlying Livewire property being true or false.

Something like 
`x-data="{
tooltip: {{ $user->disabled ? "'Enable'" : "'Disable'" }}
}"`

It works when the page loads, but it does not toggle when $user->disabled changes.

**Comment by ryangjchandler** [comment]
Hi @nijlandvpr I'll look at adding support for this. I think it should just be a case of setting the text for the tooltip when it's toggled.

**Comment by ryangjchandler** [comment]
@nijlandvpr Can you see the `x-data` property changing in the inspector?

**Comment by nijlandvpr** [comment]
@ryangjchandler Thanks for taking the time on this. Yes, my scenario is inside a Liverwire component the x-data updates but the tooltip remains as it was when the page first loads.

**Comment by ryangjchandler** [comment]
@nijlandvpr Yeah, I don't think this is a problem with the package. I'd recommend opening an issue / discussion on the Alpine or Livewire repo.

**Comment by nijlandvpr** [comment]
@ryangjchandler Sure thing - I thought I'd start with the expert on the package first. I'm also experiencing other issues with Livewire where some actions return a 404 to a "messages" route while others don't. Looks like it's losing how it recognizes DOM elements and substituting with a default - something like that anyway. New tech === "keep it simple stupid" is the message! 

Thanks

**Comment by rcubitto** [comment]
I encountered the same problem, I was able to solve it using `@entangle` and an extra property (where you should store "Enable" or "Disable")

**Comment by rcubitto** [comment]
@nijlandvpr yes, if the value of `someStringProp` changes, the tooltip gets updated accordingly.

**Comment by nijlandvpr** [comment]
> @nijlandvpr yes, if the value of `someStringProp` changes, the tooltip gets updated accordingly.

The scenario I was looking at involved a backend rendered table, with rows containing a boolean is_disabled state. Even if is_disabled was text, looping through the rows as objects or array elements wouldn't entangle - I think!

`x-data={ @entangle('user->is_disabled') }`
